extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/resources/translations/fr.json

```diff
@@ -1,27 +1,113 @@
 {
 
-    "LOGIN" : {
+  "LOGIN" : {
```

Comment:
Indentation has been heavily modified, here. In general the Guacamole code uses four space indentations, so this will need to be corrected throughout these proposed changes. Style should match was is existing within the project.

Reply:
Indentation fixed.